### ISSUE TITLE
Solid attached objects moving along surface when reloading vessel

### ISSUE THREAD

**Issue by aihaw** [ihsoft/KAS#284]
Hi! I've experienced an issue with ground base construction as I maintained two objects together with the rigid yellow attachment system.  It runs on 1.7 with the adequate version of the mod.
When reloading, the attachment system stays at the same place but the parts attached move...
![screenshot14](https://user-images.githubusercontent.com/16972795/68905797-b12d6680-0742-11ea-9299-181ba527a059.png)

I use on this base the ground tether system coming from roverdude mks maybe it makes the base buggy... It cost me some parts already as the physics don't like it.
Thank you for your consideration!

**Comment by ihsoft** [comment]
I'm confused. First of all, the "yellow attachment system" (RTS-1 part) is _not_ rigid. It was intentionally made flexible. The connected vessels can freely move relative to each other up to 30m distance.

Second, the screenshot. It shows RTS-1 part somewhat separated from the main vessel. It's hard to understand from the screenshot what's happening there. Could you please share a video?

**Comment by tjaccardi** [comment]
I have this issue as well, I've been trying to narrow it down. Basically sometimes when you timewarp parts will move a long a vessel from their original positions. The Claw ESPECIALLY triggers it and is even worse the more claws in a single scene. It also happens when I KAS attach and remove parts and doesn't seem to happen with KAS uninstalled.

**Comment by ihsoft** [comment]
If you manage to reproduce the case, could you please also capture the log? KAS attaches vessels the same way as the docking ports do. There is no difference. I still don't understand how to reproduce the case.

**Comment by tjaccardi** [comment]
Yeah I'll get a log when I have free time. I did some research a while back and found something on the squad issue tracker called the "claw bug" where people were saying the same things was happening. I could never find it again however. I'll try to get it uploaded tonight and upload the ksp/output log.

**Comment by tjaccardi** [comment]
https://drive.google.com/file/d/1A_wXQRmG5FlqFTVCKECEU6SkLh_K77R2/view?usp=sharing
https://drive.google.com/file/d/1sr-9jfITyt_pVqbF36QrAcO4ZeLcQKIx/view?usp=sharing
here you go, they were too big for pastebin.

**Comment by tjaccardi** [comment]
Sorry I overlooked reproducing. To be honest I have no idea, it happens randomly when messing with KAS by attaching and detaching. Attaching moreso, or whenever you get those freezes it seems to summon it a little more. I can give you a save file but if you want what I did: take a claw, rendezvous with an asteroid. Attach and mess with KAS by attaching garbage to the roid, craft, etc. or even try docking. To really get it going attach more than one claw and try messing with KAS and then timewarp. The claw attached craft will move as you go INTO timewarp and experience force as if they were moved upon exiting timewarp. I had this happen with a surface base to when I was removing bits and pieces.

**Comment by arturbac** [comment]
Do as follows : place 2 heavy rovers on mun (made from fuel cans one with drill second with refining) Use heavy landing gear, connect it with RTS-1, click docked and they will start moving or even flying ...
